### ISSUE TITLE
Feature request) Is it possible to increase max width of menubar item?

### ISSUE THREAD

**Issue by upstroke-version** [sindresorhus/one-thing#7]
Current max width of menubar text feels kinda short.
Is this technical limitation or artificial limit?
I have seen some apps like [Tuneful](https://github.com/martinfekete10/Tuneful) that can display longer menubar item than OneThing.
Is it possible to increase max width?
If possible maybe even provide some customizable max length option?

screen shot of my menubar. OneThing on left, Tuneful on right displaying music cover art and title.
Tuneful displays much longer text.
![1](https://github.com/user-attachments/assets/97024d83-8272-4494-bb49-307bbcaf0739)

**Comment by sindresorhus** [comment]
You can change the width in the app settings. Just click the "..." button in the window where you write text.

**Comment by upstroke-version** [comment]
That is weird...
I've installed latest version from app store and
I don't even have a "..." in editor window.
When I click the empty place where there should be a "..." button, nothing happens.
![1](https://github.com/user-attachments/assets/71ddd73b-8790-4003-a7f6-90b9db16965d)


I've rebooted and tested again in a macOS safe mode, but the  "..." button is still not there.
![2](https://github.com/user-attachments/assets/5936c73d-4a0f-4a6c-a72f-21ef7377a8ef)

I'm on a latest macOS 15.1 dev beta, so that might be the cause of issue but I'm not sure...
I will test on fresh macOS 14 vm and macOS 15 vm to see if this is issue is caused by some regression on macOS 15.
